nservicebus/wcf/index.md

```diff
@@ -1,22 +1,43 @@
 ---
 title: Exposing Endpoints via WCF
 summary: Receiving and processing messages which pass through WCF
+component: Wcf
 tags:
  - WCF
  - Hosting
 redirects:
  - nservicebus/how-do-i-expose-an-nservicebus-endpoint-as-a-web-wcf-service
+ - nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service
 related:
  - samples/web/wcf-callbacks
+ - nservicebus/messaging/callbacks
 ---
 
-Note: Starting from NServiceBus Core Versions 6 and above, the built-in support for WCF integration has been removed. For more details refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7.md).
+To handle responses on the client, the client (or the sending process) must have its own queue and cannot be configured as a SendOnly endpoint. When messages arrive in this queue, they are handled just like on the server by a message handler:
 
-Inherited from `NServiceBus.WcfService<TCommand, TErrorCode>`, as shown below. `TCommand` is the message type of the request. `TErrorCode` must be an enumerated type, and should represent the result of processing the command. Example:
+snippet:WcfEmptyHandler
+
+## Prerequisites for WCF functionality
+
+In NServiceBus Version 4 and above the WCF support has been moved to [NServiceBus.Host NuGet package](https://www.nuget.org/packages/NServiceBus.Host).
+
+In NServiceBus.Host Version 5 to 6 WCF support is built into the NuGet package.
```

Comment:
>In NServiceBus.Host Version 5 to 6 WCF support is built into the NuGet package.

Can we combine both this sentence and the next? Suggestion below: 

In Versions 5 and 6 of the NServiceBus.Host, support for WCF integrations was built into the host. Starting from Versions 7 and above, WCF support has been separated into its own nuget package, [NServiceBus.Wcf](https://www.nuget.org/packages/NServiceBus.Wcf). This package also has a dependency on [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/) and the endpoint hosting the WCF service also needs to [configure the callbacks](/nservicebus/messaging/callbacks.md) appropriately.